tutorials/focalboard-project-management/index.mdx

```diff
@@ -0,0 +1,219 @@
+content:
+  h1: Project management for technical teams with Focalboard on Ubuntu Instances
+  paragraph: This page shows how to setup a Focalboard project-management tool on Cost-Optimzed Instances running Ubuntu Linux
+tags: compute instance focalboard ubuntu project-management open-source
+hero:
+categories:
+  - compute
+  - instance
+  - build-and-maintain
+dates:
+  validation: 2023-02-22
+  posted: 2023-02-22
+---
+
+Focalboard is an open-source, multilingual, self-hosted project management tool developed by Mattermost. 
+It helps organizations to maintain a single source of truth, so teams stay aligned to complete tasks, reach milestones, and achieve their goals. 
+Focalboard provides a full suite of features such as Kanban, table, gallery, and calendar view. Filters allow staying focussed on the highest priority items. 
+The application is a privacy-caring alternative to Trello, Notion, and Asana and can be installed easily on Scaleway Instances.
+
+<Message type="tip">
+  We recommend you follow this tutorial using a [Cost-Optimized Instance](/compute/instances/reference-content/choosing-instance-type/).
+</Message>
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
+  - You have [created an Instance](/compute/instances/how-to/create-an-instance/) that runs Ubuntu Jammy Jellyfish (22.04 LTS)
+  - You have a (sub)domain pointed to the IP address of your Instance (A-record)
+</Message>
+
+## Installation of the required software
+
+In the following steps, we will make sure that the software and the required packages installed on your Instance are up-to-date, and that you have downloaded and unpacked the Focalboard application.
+
+1. Log into your Instance using SSH:
+    ```
+    ssh root@<your_scaleway_instance_ip>
+    ```
+2. Update the `apt` package manager and upgrade the installed software on your Instance to the latest version.
+    ```
+    apt update && apt upgrade -y
+    ```
+3. Install the required software packages using the `apt` package manager.
+    ```
+    apt install nginx python3-certbot-nginx mariadb-server ufw
+    ```
+4. Download and unpack Focalboard. In this tutorial, we use v7.8.2 of Focalboard, but you can find the **lates**t release in the project's [GitHub repository](https://github.com/mattermost/focalboard/releases).
+    ```
+    wget https://github.com/mattermost/focalboard/releases/download/v7.8.2/focalboard-server-linux-amd64.tar.gz
+    tar -xvzf focalboard-server-linux-amd64.tar.gz
+    mv focalboard /opt
+    ```
+
+## Nginx configuration
+
+We use Nginx aas a proxy in front of the application to prevent it from direct access and by providing an additional layer of security through TLS encryption.
```

Comment:
```suggestion
We use Nginx as a proxy in front of the application to prevent it from direct access and by providing an additional layer of security through TLS encryption.
```